<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Reminder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="placeholderText">
         <string>Search events...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortCombo">
        <item>
         <property name="text">
          <string>Sort: Asc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sort: Desc</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add Event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove Event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeMultipleButton">
        <property name="text">
         <string>Remove Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionToggle_Autostart"/>
    <addaction name="actionShow_MainWindow"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <connections/>
  <action name="actionToggle_Autostart">
   <property name="text">
    <string>Enable Auto-start</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionShow_MainWindow">
   <property name="text">
    <string>Show Main Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_searchEdit_textChanged(QString)</slot>
  </connection>
  <connection>
   <sender>sortCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_sortCombo_currentIndexChanged(int)</slot>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_addButton_clicked()</slot>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_removeButton_clicked()</slot>
  </connection>
  <connection>
   <sender>removeMultipleButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_removeMultipleButton_clicked()</slot>
  </connection>
  <connection>
   <sender>actionToggle_Autostart</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionToggle_Autostart_triggered()</slot>
  </connection>
  <connection>
   <sender>actionShow_MainWindow</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionShow_MainWindow_triggered()</slot>
  </connection>
 </connections>
</ui>
